admin/css/views/status.css

```
@@ -331,10 +331,24 @@
+/*
+ * 内容层：与视觉面同步上浮。
+ * 小字号子数据源在 GPU 合成层上容易糊，因此：
+ * 1) 不用常驻 translateZ / will-change（避免平时就进合成层）
+ * 2) hover 时用 translate3d 整像素位移
+ * 3) 子项 .connection-sub-source-item 禁止再挂 transform/will-change
+ */
 .connection-item > * {
     transition: transform 0.2s cubic-bezier(0.2, 0.8, 0.2, 1);
+    -webkit-font-smoothing: antialiased;
+    -moz-osx-font-smoothing: grayscale;
+    text-rendering: geometricPrecision;
```

Stylelint (17.14.0)

[error] Expected "geometricPrecision" to be "geometricprecision" (value-keyword-case)

(value-keyword-case)